pkg/eventpersisterdwh/persister/evaluation_event.go

```diff
@@ -220,5 +224,16 @@ func (w *evalEvtWriter) listExperiments(
 		handledCounter.WithLabelValues(codeFailedToListExperiments).Inc()
 		return nil, err
 	}
-	return exp.([]*exproto.Experiment), nil
+	// Filter the stopped experiments
+	// Because the evaluation and goal events may be sent with a delay for many reasons from the client side,
+	// we still calculate the results for two days after it stopped.
+	now := time.Now().In(w.location)
+	experiments := make([]*exproto.Experiment, 0, len(exp.([]*exproto.Experiment)))
+	for _, e := range exp.([]*exproto.Experiment) {
+		if e.Status == exproto.Experiment_STOPPED && now.Unix()-e.StopAt > 2*day {
```

Comment:
I implemented the same logic used in the experiment calculator.

https://github.com/bucketeer-io/bucketeer/pull/395/files#diff-b0f1a6e9d024e46f089e3bc6573a1c8f6dfaf821d6b3898e3c06794bbc4d0dd8R128-R133